tests/test_workflow.py

```diff
@@ -252,6 +252,22 @@ def test_preprocess(self, preprocessed_dir_pl: Path):
         output_path = preprocessed_dir_pl / "2020-01-01_fluxes_storages.nc"
         assert output_path.exists()
 
+        # verify outputs
+        expected_path = Path(
+            "tests/test_data/verify_output/preprocessed_data/2020-01-01_fluxes_storages.nc"
```

Comment:
It is not clear from the filename/path that this is reference data for pressure levels. Should we make that more explicit?

Reply:
That's slightly challenging: we want to make it easy to overwrite the regression data with new data. If we change the filenames, the output filenames won't be the same anymore either. So instead of just overwriting the regression test files, you have to copy the new one, delete the old one, and rename the new one.